Clarify frontend alias location (#3664)
vaadin/docs: articles/hilla/lit/guides/endpoints.adoc

.Frontend Directory Alias
[NOTE]
====

> The `'Frontend/'` path prefix is an alias for the `{project.basedir}/src/main/frontend` directory in your application project (or `{project.basedir}/frontend` if using the default convention from before Vaadin 24.4).

Reviewer: ```suggestion
The 'Frontend/' path prefix is an alias for the `{project.basedir}/src/main/frontend` directory in your project. If you are using the default convention from before Vaadin 24.4, it refers to `{project.basedir}/frontend`.
```
